### DISCUSSION TITLE
Beginner-Friendly Tutorial and Documentation to Promote the Use of NASA Open MCT

### DISCUSSION THREAD

**Discussion by MarkCodering** [nasa/openmct#5336] [Polls]
Hi there,

As a user and developer of the NASA Open MCT application for years, I found it could be difficult for beginners to this framework to build and test an application developed with NASA Open MCT. Therefore, I proposed to create beginner-friendly documentation or tutorial while promoting the use of NASA Open MCT in colleges worldwide. 
 
Target Audience: 
- Student Team for building a sounding rocket, SAE race car, etc.
- Academic Departments with no or few computer science backgrounds

How to?
- Explain the documentation and tutorial in a much simple terms
- Use the graphic presentation to explain abstract concepts in the framework

**Comment by marcelo-earth** [comment]
I support this because the previous documentation graphics to explain the core of OpenMCT was really helpful:

![](https://user-images.githubusercontent.com/47821093/171047755-e6d6ce92-74ae-42ad-b23a-ff512dcae485.png)

**Comment by Kokodora** [comment]
Hey, I thought the same! Did you know about this series:
https://github.com/nasa/openmct/discussions/4894

**Comment by koyuul** [comment]
Thread is 2 years old, but just wanted to say that based on my experience, I also think the documentation should be touched up a bit more. The tutorial still has some deprecated time commands in it, and I felt that the time API documentation in general was a bit hard to work with and understand. 

Could just be me, but I feel like the relationships between the clock, time system, bounds, clock offsets could be defined clearer (maybe with a diagram or something). Also on that note, I feel like the examples such as the [defining and registering clocks example](https://github.com/nasa/openmct/blob/master/API.md#defining-and-registering-clocks) would work better if they included actual example functions instead of leaving a comment showing where the working function would be. I know that the actual implementations would vary in a real application, but a concretely implemented function example would be helpful in seeing what your own should look like.

Don't mean to be rude or anything, I am enjoying using it overall but think updating the docs would be helpful :smile:

**Comment by pdelv** [comment]
I think it would be great!

**Comment by akhenry** [comment]
@shefalijoshi
